### DISCUSSION TITLE
SEB - Glassbrick

### DISCUSSION THREAD

**Discussion by JSCSF** [SafeExamBrowser/seb-win-refactoring#153] [Ideas]
Hi guys

You may or may not be familiar with a program named Glassbrick. If not, its a really good tool that allows students to magnify the screen (exactly like a magnifying glass), as well as inverting / saturating / tinting the colours on screen. Its extremely useful to students that suffer from learning difficulties such as dyslexia / dyspraxia, and makes exams so much more accessible to these students.

I have managed to add Glassbrick as a permitted process via the SEB config tool so that these functions can be used through SEB. It is however a time consuming process when doing this on numerous computers, and it is difficult to turn inverted colours on or off when using SEB in full screen mode.

I wondered if you had ever considered adding these functions to SEB, or making it easier to run a program like this whilst also running SEB in full screen mode.

Almost 2% of people in the UK have a diagnosed learning difficulty and would benefit from these functions, and even more live their lives undiagnosed. It would really help make the platform more accessible to these students.

Just thought I would raise the idea and gain your thoughts!

Thanks.

James.

**Comment by dbuechel** [comment]
Thanks for your input. We have accessibility features on [our roadmap](https://safeexambrowser.org/about_roadmap_en.html), but nothing like Glassbrick thus far. If you are member of an institution or alike, you might want to consider joining the [SEB Alliance](https://safeexambrowser.org/alliance/members.html) to help with sustaining the development of SEB, as accessibility features in particular are time consuming and costly.

Regarding the configuration procedure you mentioned: I am not entirely certain, but it appears like you have been adding Glassbrick manually with the Configuration Tool on each computer. If so, there is indeed an easier way, in that you can simply create a configuration file once and then distribute it to numerous computers, see also https://safeexambrowser.org/windows/win_usermanual_en.html#ConfigFilePane.
